package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "promptxchange",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "next": "^12.0.7",
```

Comment:
do we need next ?

Reply:
Not really. It's what have been using recently so I decided to continue with it. Should I switch to React?